collector/lib/ProcfsScraper.cpp

```diff
@@ -565,32 +566,22 @@ bool ConnScraper::Scrape(std::vector<Connection>* connections, std::vector<Conta
   return ReadContainerConnections(proc_path_.c_str(), process_store_, connections, listen_endpoints);
 }
 
-Process ProcessScraper::ByPID(uint64_t pid) {
+bool ProcessScraper::Scrape(uint64_t pid, ProcessInfo& pi) {
```

Comment:
pi makes me think of 3.1415... Maybe change to processInfo.

Reply:
:heavy_check_mark:  Renamed to something more explicit.